addon/helpers/or.js

```diff
@@ -0,0 +1,11 @@
+import {helper} from '@ember/component/helper';
+
+export function or(params) {
+  let bool = false;
+  params.forEach((param) => {
+    bool = bool || param;
+  });
+  return bool;
+}
+
+export default helper(or);
```

Comment:
Can we use https://www.npmjs.com/package/ember-truth-helpers instead of writing our own?

Reply:
Note that it's a new open-sourced addon, but still I guess we can if the package above is not too big ( I tried to add such an handlebar helper library few years ago and was told at the time I shouldn't - so that stick). Adding to the bullet list now for next PRs.